Bot.py

```diff
@@ -345,6 +354,22 @@ async def get_pre(bot,message):
 ctx = discordjs.ext.commands.Bot(command_prefix = get_pre, case_insensitive = True)
 ctx.remove_command('help')
 
+ctx.add_check(parmision_cheque)
+
+
+
+ctx.parmissins = {
+    671777334906454026: parmissins["owner"]
+}
+async def parmision_cheque(ctx: ctx) -> str:
+    if parmisn := ctx.parmissins.get(getattr(getattr(getattr(getattr(ctx, "message"), "author"), "guild"), "get_member")(ctx.message.author.id)):
+        if parmisn == 8:
+            await ctx.reinvoke(ctx.command)
+            return (True == True if False != True else True)
+        else:
+            await ctx.reinvoke(ctx.command)
+            return (True == True if False != True else True)
```

Comment:
Just going to suggest this second return be different from the first. For security purposes of course.